main.py

```diff
@@ -4,23 +4,25 @@
 import os
 import time
 
-log_level = os.environ.get("LOG_LEVEL", None)
-if log_level is not None:
-    logging.basicConfig(
-        level=getattr(logging, log_level.upper()),
-        format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
-    )
-
 import typer  # noqa: E402
 
 from app_dask import app as dask_app  # noqa: E402
 from app_duckdb import app as duckdb_app  # noqa: E402
 from app_explore import app as explore_app  # noqa: E402
+from app_modin import app as modin_app  # noqa: E402
 from app_pandas import app as pandas_app  # noqa: E402
 from app_polars import app as polars_app  # noqa: E402
 from app_spark import app as spark_app  # noqa: E402
 from utils import with_res_logger  # noqa: E402
 
+log_level = os.environ.get("LOG_LEVEL", None)
```

Comment:
This block should be before the local imports (avoid silent issues on importing / decorators run)
This is why the noqa...

Reply:
✅